### ISSUE TITLE
Train the FTM Network

### ISSUE THREAD

**Issue by Orangessssss** [QiaolingChen00/FTM#2]
Can you elaborate on how the GID dataset is assigned the training set, validation set when training the FTM Network?
In the paper, you mentioned that Set-C was divided into 150 training and 30 vvalidation. set-F train/val/testing 1500/3750/750 respectively.
Can you describe how your folder structure, Data->gid->(train/val), is arranged?

**Comment by Orangessssss** [comment]
@Chenqll
